src/Autofac.Integration.Web/Autofac.Integration.Web.csproj

```diff
@@ -1,113 +1,66 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{4BDD9D14-BACA-416C-919E-E9FF100F8F5D}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Autofac.Integration.Web</RootNamespace>
-    <AssemblyName>Autofac.Integration.Web</AssemblyName>
-    <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\Autofac.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <TargetFrameworkProfile />
+    <TargetFramework>net461</TargetFramework>
+    <AssemblyTitle>Autofac.Integration.Web</AssemblyTitle>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <Description>ASP.NET web forms integration for Autofac.</Description>
+    <Copyright>Copyright © 2014 Autofac Contributors</Copyright>
+    <Version>4.0.0.0</Version>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <FileVersion>4.0.0.0</FileVersion>
```

Comment:
Please remove `Version`, `AssemblyVersion` and `FileVersion`.
You can add `VersionPrefix` same as [here](https://github.com/autofac/Autofac.Extensions.DependencyInjection/blob/develop/src/Autofac.Extensions.DependencyInjection/Autofac.Extensions.DependencyInjection.csproj#L5) for instance and update the `.appveyor.yml` accordingly.